featuretools/tests/conftest.py

```diff
@@ -136,37 +137,40 @@ def pd_diamond_es():
         ('customers', 'id', 'transactions', 'customer_id'),
     ]
     return ft.EntitySet(id='ecommerce_diamond',
-                        entities=entities,
+                        dataframes=dataframes,
                         relationships=relationships)
 
 
 @pytest.fixture
 def dask_diamond_es(pd_diamond_es):
-    entities = {}
-    for entity in pd_diamond_es.entities:
-        entities[entity.id] = (dd.from_pandas(entity.df, npartitions=2), entity.index, None, entity.variable_types)
+    dataframes = {}
+    for df in pd_diamond_es.dataframes:
+        dataframes[df.ww.name] = (dd.from_pandas(df, npartitions=2), df.ww.index, None, df.ww.logical_types, get_df_tags(df))
 
-    relationships = [(rel.parent_dataframe.id,
+    relationships = [(rel.parent_dataframe.ww.name,
                       rel.parent_column.name,
-                      rel.child_dataframe.id,
+                      rel.child_dataframe.ww.name,
                       rel.child_column.name) for rel in pd_diamond_es.relationships]
 
-    return ft.EntitySet(id=pd_diamond_es.id, entities=entities, relationships=relationships)
+    return ft.EntitySet(id=pd_diamond_es.id, dataframes=dataframes, relationships=relationships)
 
 
 @pytest.fixture
 def ks_diamond_es(pd_diamond_es):
     ks = pytest.importorskip('databricks.koalas', reason="Koalas not installed, skipping")
-    entities = {}
-    for entity in pd_diamond_es.entities:
-        entities[entity.id] = (ks.from_pandas(pd_to_ks_clean(entity.df)), entity.index, None, entity.variable_types)
-
-    relationships = [(rel.parent_dataframe.id,
+    dataframes = {}
+    for df in pd_diamond_es.dataframes:
+        semantic_tags = {}
+        for col_name in df.columns:
+            semantic_tags[col_name] = df.ww.semantic_tags[col_name] - {'time_index', 'index'}
```

Comment:
Is this necessary with the usage of `get_df_tags` below?

Reply:
whoopse meant to remove!